### ISSUE TITLE
Error when enabling Detailed Billing

### ISSUE THREAD

**Issue by gitlab-pano** [Scalr/scalr#76]
The error is on the bucket name field, but I am sure the name is correct.

Cannot access billing bucket with name my-bucket-name. Error: AWS Error: Request GetUser failed. Credential should be scoped to a valid region, not US-West-2.  The scalr server is running in US-West-2. (I originally installed it in another region then moved it to US-West-2). Is there somewhere in the configuration that I need to change the region?

**Comment by gitlab-pano** [comment]
This is still a problem.  The same exact settings work fine in the hosted version.

**Comment by recipe** [comment]
@gitlab-pano This issue was fixed in Open Source Scalr 5.11.22. Cloud Credentials UI allows to select correct AWS cloud location for the detailed billing bucket http://i.imgur.com/xqF4qKK.png

**Comment by masterxavierfox** [comment]
Hello @gitlab-pano did you ever solve this problem? im having the same problem

**Comment by gitlab-pano** [comment]
Yes I did. It had to do with setting the correct region. Both when creating the billing bucket and when setting up the application. Both have to be the same. Us-east-1
